src/theme/theme.js

```diff
@@ -0,0 +1,5 @@
+import { defaultColors } from '../constants/color';
+
+export const defaultTheme = {
+  defaultColors,
+};
```

Comment:
여기서도 테마를 하나의 타입으로 보는게 좋겠지?

```typescript
interface IColorTheme {
  primary1: string;
  primary2: string;
  primary3: string;
  black: string;
  gray1: string;
  gray2: string;
  gray3: string;
  gray4: string;
  gray5: string;
  gray6: string;
  gray7: string;
  white: string;
  red1: string;
}
```

이렇게 interface 정의해주고

```suggestion
export const defaultColorTheme: IColorTheme = {
  primary1: '#6129FF',
  primary2: '#7A59FF',
  primary3: '#A697FF',
  black: '#000000',
  gray1: '#2C2A32',
  gray2: '#555561',
  gray3: '#9796A0',
  gray4: '#CFCED4',
  gray5: '#DFDFE7',
  gray6: '#EFEEF3',
  gray7: '#F7F7F8',
  white: '#ffffff',
  red1: '#FF3232',
}
```

이런식으로 적어두면 다른 테마를 만들때도 저 필드들을 그대로 가져갈 수 있음!

Reply:
생각해보니 `constants/colors.ts` 에 넣을 필요도 없고 여기에 그냥 다 넣어도 되겠다
그리구 `themes/theme.ts` 보다는 `themes/index.ts`로 하는게 import 할 때 훨씬 깔끔할 것 같어

이렇게 import 할 수 있잖아
```ts
import { defaultColorTheme } from "src/themes/theme"; // old
import { defaultColorTheme } from "src/themes"; // new
```

import 할때 깔끔해지기도 하고 `index.ts/js` 파일들은 별다른 언급 없으면 상위 폴더를 임포트했을때 기본값으로 가져와지게됨!